classes/services/PKPSubmissionService.inc.php

```diff
@@ -47,6 +47,8 @@ public function __construct() {
 	 * 		@option string searchPhrase
 	 * 		@option int count
 	 * 		@option int offset
+	 *		@option string returnObject Whether to return submission or published
```

Comment:
Generally I prefer `define`d constants for this kind of thing, rather than hard-coded strings. Open to other approaches too if you think there are good reasons.

Reply:
No that's best. Just an old (bad) habit from WordPress.